db/pre-re/refine_db.conf

```diff
@@ -0,0 +1,125 @@
+/********************************************************************************
+ *			Pre-Renewal Refine Database				*
+ ********************************************************************************
+Armor/WeaponLevel1~4: { (0 - Armors, 1 - Level 1 Weapons, 2 - Level 2 Weapons, 3 - Level 3 Weapons, 4 - Level 4 Weapons)
+						- For armors, values of 100 add 1 armor defense.
+						- For weapons, values of 100 add 1 ATK.
+	StatsPerLevel: (int)			This value is applied for ever level.
+	RandomBonusStartLevel: (int)		This value specifies the start point for those levels that give a random bonus value (usually the first unsafe upgrade).
+	RandomBonusValue: (int)			A random number between 0 and (Random bonus start level - Upgrade level + 1) * this value is applied for all upgrades past.
+						Random bonus start level. This is only applied for weapons, and not displayed client-side.
+	Rates: {				Per level configuration of the refine rates.
+		Lv1~10: {			Level of refine
+			Chance: (int)			Chance of successful refine (100 = 100%)
```

Comment:
From what I see below, chance also is optional (and defaults to 100) - it'd be nice to have it mentioned here